lingua_franca/config.py

```diff
@@ -1 +1,2 @@
 load_langs_on_demand = False
+inject_timezones = True
```

Comment:
This is probably covered better in the overall review comment.

Reply:
not setting the default_tz falls back to system tz or UTC as last resort. but internally it still ensures all datetime objects are tz aware

setting `inject_timezones = False` skips the tz attachment completely, so it is kinda like the old behavior (except for the `now_local` not being used bug). This makes the returned date timezone undefined/depend on input data like it currently is.

Reply:
there is no config yet as said in previous comment, so timezone should not be defined here yet

by using wrapper methods is trivial to add a proper config later and point the method to it + update docs to deprecate method. adding a bunch of stuff to placeholder module that is essentially a name will likely cause trouble down the road.

for the purpose of this PR lets pretend config module does not exist yet, `inject_timezones = True` can be completely removed if you want, as mentioned it was mostly meant as a backwards compat flag and shouldnt really be set by end users 99.9% of the time

Reply:
sry, missed the main question, UTC vs system comes from `tzlocal() `documentation, it is there only for backwards compat and because thats the current mycroft-core default (if mycroft.conf is not set it uses `tzlocal()` )

Reply:
OK.  Resolving this conversation as it will be addressed in a later revision.